### ISSUE TITLE
Alt Text and Title

### ISSUE THREAD

**Issue by farishkash** [Tonyyyy1234/Dasmoto_Project#2]
https://github.com/Tonyyyy1234/Dasmoto_Project/blob/f6c014120d31d76e7e64ce8f79e28433e090f28d/Dasmoto's/index.html#L34

Good use of alt text.  I wanted to pass on some information on title and why it really isn't used.

`Relying on the title attribute is currently discouraged as many user agents do not expose the attribute in an accessible manner as required by this specification (e.g., requiring a pointing device such as a mouse to cause a tooltip to appear, which excludes keyboard-only users and touch-only users, such as anyone with a modern phone or tablet).`

Basically, using title does nothing on mobile devices because there is no value since there is no pointer.  Screen readers focus primarily on the alt text and exclude the title.
